addon/-task-state-mixin.js

```diff
@@ -50,5 +57,18 @@ export default Mixin.create({
 
   _scheduler: null,
 
+  _resetState() {
+    this.setProperties({
```

Comment:
Shouldn't this reset `lastStarted` too?

Reply:
Oops, missed this this... yes it probably should. I'll make this adjustment on a separate commit and release in the next version.